app/cli/cmd/workflow_create.go

```diff
@@ -51,6 +51,11 @@ func newWorkflowCreateCmd() *cobra.Command {
 				// Try to find it by name
 				c, err := action.NewWorkflowContractDescribe(actionOpts).Run(contractRef, 0)
 				if err != nil || c == nil {
+					_, err := action.LoadFileOrURL(contractRef)
```

Comment:
this is just to improve this other command error message, not specifically about this problem.